package.json

```diff
@@ -94,7 +94,8 @@
     "@react-navigation/material-top-tabs": "5.2.19",
     "@react-navigation/native": "5.7.6",
     "@react-navigation/stack": "5.9.3",
-    "@segment/analytics-react-native": "1.5.0",
+    "@segment/analytics-react-native": "2.3.2",
+    "@segment/sovran-react-native": "0.2.8",
```

Comment:
Do we need this one as a dependency?

Reply:
This `sovran` is some kind of a state management developed by segment. They use id under the hood to probably pass some native related data and events like `Application Opened` `Application Installed` etc. The rest like sending events got moved to JS. SO I would leave it since this is what they advise to install in the docs itself